pkgs/top-level/all-packages.nix

```diff
@@ -15133,6 +15177,22 @@ with pkgs;
     meta.broken = stdenv.hostPlatform.isDarwin;
   });
 
+  # Does not build yet:
+  #   ld: warning: libgcc_s.so.1, needed by .../libgo/.libs/libgo.so,
+  #     not found (try using -rpath or -rpath-link)
+  #   libgo.so: undefined reference to `_Unwind_RaiseException@GCC_3.0'
```

Comment:
I think we can still uncomment this, the failure is the same as for `gccgo12`...

Reply:
Sounds reasonable. Uncommented `gccgo13` and updated commit messages.